tests/tests.py

```diff
@@ -1,26 +1,44 @@
+import os, sys
 import logging
 
 import matplotlib
-matplotlib.use('Agg')
+if os.environ.get('DISPLAY','') == '':
+    logging.warning('No display found. Using non-interactive Agg backend')
+    matplotlib.use('Agg')
 
 import numpy as np
-from scipy.misc import imread
+from scipy.ndimage import imread
 from matplotlib import pyplot as plt
 
-import morphsnakes
+sys.path += [os.path.abspath('.'), os.path.abspath('..')]  # Add path to root
+from morphsnakes.morphsnakes import MorphGAC, MorphACWE
+from morphsnakes.morphsnakes import gborders, evolve_visual, evolve_visual3d
+import morphsnakes.multi_snakes as multi_ms
 
-PATH_IMG_NODULE = 'images/mama07ORI.bmp'
-PATH_IMG_STARFISH = 'images/seastar2.png'
-PATH_IMG_LAKES = 'images/lakes3.jpg'
-PATH_ARRAY_CONFOCAL = 'images/confocal.npy'
```

Comment:
Why do you remove this?

Reply:
is not removed, just it uses os.path.join(...) like
`PATH_IMG_NODULE = os.path.join(PATH_IMAGES, '...')`